### DISCUSSION TITLE
Open URL in new tab with CMD-enter (or similar)

### DISCUSSION THREAD

**Discussion by Coldkennels** [minbrowser/min#2517] [Ideas]
On most browsers, you can hold a modifier key (such as CMD on Mac) when pressing enter after typing a URL to open it in a new tab instead of the current one.

Currently, in Min, CMD-enter does nothing; the "keyboard shortcuts" page (https://github.com/minbrowser/min/wiki#keyboard-shortcuts) suggests it should autocomplete a URL with www. and .com, but that doesn't seem to exist in the browser currently. Also, I can't see an "open URL in new tab" option in the Keyboard Shortcuts panel in min://settings, so this functionality can't be added in by the end user. 

Honestly, I didn't even realise how much I'd got used to CMD-enter until I swapped to Min - it was one of the biggest usability hurdles in moving to Min for me.

**Comment by PalmerAL** [comment]
Huh, this feature dates back to #138 (current implementation [here](https://github.com/minbrowser/min/blob/master/js/searchbar/searchbar.js#L69-L81)). The request was "ctrl + enter integration"; I interpreted that as meaning ctrl or cmd, depending on the platform. But I see now that other mac browsers are doing what you describe - ctrl adds ".com", cmd opens in a new tab.

Do you know what other browsers do on Windows, where there's no cmd key?
